### DISCUSSION TITLE
ident-blankline.nvim

### DISCUSSION THREAD

**Discussion by franticflux** [NvChad/NvChad#3072] [Q&A]
I just updated all plugins and for some reason indent-blankline stopped working. I reverted back to the slightly older version of that particular plugin by git checkout d343409a9f7ce4573955a181a0dfb71a103040d3 in directory of the plugin, ie ..lazy\indent-blankline.nvim\ and everything is fine. I'm on Windows 11, no wsl, so maybe there is something wrong with the paths as usual...

I don't have time to investigate further right now and don't know if its a problem with my config, nvchad or indent-blankline, but just wanted to share my workaround in case someone else stumbles upon this.  The breaking change for me is:

```
commit 04e44b09ee3ff189c69ab082edac1ef7ae2e256c (HEAD, tag: v3.8.3, origin/master, origin/HEAD, master)
Author: Maria José Solano <majosolano99@gmail.com>
Date:   Sat Oct 26 19:41:19 2024 -0700

    feat: use faster version of validate if available
```

**Comment by franticflux** [comment]
Update: Apparently this is now a known issue with indent-blankline, read more [indent-blankline issue 936, github](https://github.com/lukas-reineke/indent-blankline.nvim/issues/936). 

The recommended solution is to update to the latest version (the nightly build) of Neovim and it works for me on on my freshly compiled

NVIM v0.11.0-dev-1065+gff93cccbc1
Build type: Release
LuaJIT 2.1.1727870382

**Reply by siduck** [reply]
@franticflux i updated to the latest, facing no issues
